repos/system_upgrade/common/actors/setsystemdservicesstate/libraries/setsystemdservicesstate.py

```diff
@@ -0,0 +1,26 @@
+from leapp.libraries.stdlib import api, CalledProcessError, run
+from leapp.models import SetSystemdServicesStateTask
+
+
+def _try_set_unit_state(command, unit):
```

Comment:
having service in the upper function and unit in this function is little bit confusing. stay consistent